### DISCUSSION TITLE
How to set $EDITOR environment variables on Windows

### DISCUSSION THREAD

**Discussion by WakerCN** [SBoudrias/Inquirer.js#1273] [General]
when use type:'editor' prompt,
I want to use vim on Windows, how to set $EDITOR environment 🤣？

**Comment by SBoudrias** [comment]
Maybe this will help https://superuser.com/questions/949560/how-do-i-set-system-environment-variables-in-windows-10

**Reply by WakerCN** [reply]
thank u
